### ISSUE TITLE
Cluster on GCP connection cannot be established with v0.8.0-pro

### ISSUE THREAD

**Issue by skeeey** [submariner-io/submariner#992]
**What happened**:
I have three clusters `cluster-aws`, `clsuter-gcp` and `clsuter-azure`, and the Submariner broker is installed in `cluster-aws`

```
# cluster-aws
./subctl show all

Showing information for cluster "wlawscluster":
    Discovered network details:
        Network plugin:  OpenShiftSDN
        Service CIDRs:   [172.30.0.0/16]
        Cluster CIDRs:   [10.128.0.0/14]

CLUSTER ID                    ENDPOINT IP     PUBLIC IP       CABLE DRIVER        TYPE            
cluster-aws                   10.0.113.100    34.223.48.90    strongswan          local           
cluster-azure                 10.0.32.4       40.88.217.122   strongswan          remote          
cluster-gcp                   10.0.32.2       34.74.7.165     strongswan          remote          

GATEWAY                         CLUSTER                 REMOTE IP       CABLE DRIVER        SUBNETS                                 STATUS          
wlazurecluster-pmfkq-worker-eas cluster-azure           10.0.32.4       strongswan          172.32.0.0/16, 10.140.0.0/14            connecting      
wlgcpcluster-fkfsr-worker-b-cd9 cluster-gcp             10.0.32.2       strongswan          172.31.0.0/16, 10.132.0.0/14            connecting      

NODE                            HA STATUS       SUMMARY                         
ip-10-0-113-100                 active          0 connections out of 2 are established

COMPONENT                       REPOSITORY                                            VERSION         
submariner                      quay.io/submariner                                    0.8.0-pre0      
submariner-operator             quay.io/submariner                                    0.8.0-pre0      
service-discovery               quay.io/submariner                                    0.8.0-pre0
```

```
# cluster-gcp
./subctl show all                                                                   

Showing information for cluster "wlgcpcluster":
    Discovered network details:
        Network plugin:  OpenShiftSDN
        Service CIDRs:   [172.31.0.0/16]
        Cluster CIDRs:   [10.132.0.0/14]

CLUSTER ID                    ENDPOINT IP     PUBLIC IP       CABLE DRIVER        TYPE            
cluster-gcp                   10.0.32.2       34.74.7.165     strongswan          local           
cluster-aws                   10.0.113.100    34.223.48.90    strongswan          remote          
cluster-azure                 10.0.32.4       40.88.217.122   strongswan          remote          

GATEWAY                         CLUSTER                 REMOTE IP       CABLE DRIVER        SUBNETS                                 STATUS          
ip-10-0-113-100                 cluster-aws             10.0.113.100    strongswan          172.30.0.0/16, 10.128.0.0/14            error           
wlazurecluster-pmfkq-worker-eas cluster-azure           10.0.32.4       strongswan          172.32.0.0/16, 10.140.0.0/14            connecting      

NODE                            HA STATUS       SUMMARY                         
wlgcpcluster-fkfsr-worker-b-cd9 active          0 connections out of 2 are established

COMPONENT                       REPOSITORY                                            VERSION         
submariner                      quay.io/submariner                                    0.8.0-pre0      
submariner-operator             quay.io/submariner                                    0.8.0-pre0      
service-discovery               quay.io/submariner                                    0.8.0-pre0 
```

When I use v0.7.0, the connection between `cluster-aws` and `clsuter-gcp` can be established, but with v0.8.0-pro, it fails

I check the gateway logs, it seems no error occurred and the cables are installed successfully, see my attachment

```
# aws gateway
I1203 01:51:03.570778       1 cableengine.go:150] Successfully installed Endpoint cable "submariner-cable-cluster-gcp-10-0-32-2" with remote IP 34.74.7.165
I1203 01:51:03.570795       1 tunnel.go:63] Tunnel controller successfully installed Endpoint cable submariner-cable-cluster-gcp-10-0-32-2 in the engine
```

```
#gcp gateway
I1203 01:51:03.483628       1 cableengine.go:101] Installing Endpoint cable "submariner-cable-cluster-aws-10-0-113-100"
I1203 01:51:03.483586       1 pinger.go:48] CableEngine HealthChecker started pinger for IP "10.130.2.1"
I1203 01:51:03.487320       1 cableengine.go:150] Successfully installed Endpoint cable "submariner-cable-cluster-aws-10-0-113-100" with remote IP 34.223.48.90
I1203 01:51:03.487347       1 tunnel.go:63] Tunnel controller successfully installed Endpoint cable submariner-cable-cluster-aws-10-0-113-100 in the engine
```

**What you expected to happen**:
The connection can be established

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Submariner version (use `subctl version`): v0.8.0-pro
- Kubernetes version (use `kubectl version`): ocp 4.6.4/kuberntest 1.19
- Cloud provider or hardware configuration:
   I configured the GCP firewall with below rules
  ```
  gcloud compute firewall-rules create "submariner-udp-in-500" --network=mycluster-network --allow=udp:500 --direction=IN
  gcloud compute firewall-rules create "submariner-udp-in-4500" --network=mycluster-network --allow=udp:4500 --direction=IN
  gcloud compute firewall-rules create "submariner-udp-in-4800" --network=mycluster-network --allow=udp:4800 --direction=IN

  gcloud compute firewall-rules create "submariner-udp-out-500" --network=mycluster-network --allow=udp:500 --direction=OUT
  gcloud compute firewall-rules create "submariner-udp-out-4500" --network=mycluster-network --allow=udp:4500 --direction=OUT
  gcloud compute firewall-rules create "submariner-udp-out-4800" --network=mycluster-network --allow=udp:4800 --direction=OUT

  ```
- OS (e.g: `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Install tools:
- Network plugin and version (if this is a network-related bug):
- Others:

**Comment by skeeey** [comment]
[gateway.0.8.0.aws.log](https://github.com/submariner-io/submariner/files/5633521/gateway.0.8.0.aws.log)
[gateway.0.8.0.gcp.log](https://github.com/submariner-io/submariner/files/5633522/gateway.0.8.0.gcp.log)

**Comment by sridhargaddam** [comment]
From the Gateway logs we can see that there is some connectivity issue. Its mostly due to ACL/SecurityGroups blocking the traffic, but you mentioned that you configured the rules already. Wondering what could be missing. @mangelajo any thoughts?

`05[NET] sending packet: from 10.0.113.100[500] to 34.74.7.165[500] (500 bytes)
15[IKE] retransmit 5 of request with message ID 0
15[NET] sending packet: from 10.0.113.100[500] to 34.74.7.165[500] (500 bytes)
11[IKE] giving up after 5 retransmits
11[IKE] peer not responding, trying again (4/0)`

**Comment by skeeey** [comment]
this problem is solved with v0.8.0-rc0
